weather_mv/loader_pipeline/bq.py

```diff
@@ -193,6 +209,7 @@ def to_row(it: t.Dict) -> t.Dict:
             row[DATA_IMPORT_TIME_COLUMN] = import_time
             row[DATA_URI_COLUMN] = uri
             row[DATA_FIRST_STEP] = first_time_step
+            row[S2_LOCATION] = fetch_s2_location(row['latitude'], row['longitude'])
```

Comment:
Have you tested this e2e with a BQ database? Does this work? May I see the resulting table with live data?